src/pages/open-graph/[...path].ts

```diff
@@ -14,7 +14,8 @@ import { readFile } from 'fs/promises';
 // combo to extract title & description from each page.
 
 /** Paths for all of our Markdown content we want to generate OG images for. */
-const paths = await glob('src/pages/**/*.md');
+// @ts-expect-error 2339 - env does not exist in import.meta as this file is not included in tsconfig.json
+const paths = !import.meta.env.SKIP_OG ? await glob('src/pages/**/*.md') : [];
```

Comment:
We should be able to use `process.env` instead of `import.meta.env` and avoid the need for the `@ts-expect-error` comment.

```suggestion
const paths = process.env.SKIP_OG ? [] : await glob('src/pages/**/*.md');
```